bin/test.sh

```diff
@@ -233,14 +229,62 @@ savemv modbat.examples.NioSocket1-root-StateGraph.dot nio-path-state-500.dot
 checkfile nio-path-state-500.dot
 savemv modbat.examples.NioSocket1-root-PointGraph.dot nio-path-point-500.dot
 checkfile nio-path-point-500.dot
-# path coverage 1000 test cases
+## path coverage 1000 test cases
 run 0 $APP -n=1000 -s=1 --no-redirect-out --dotify-path-coverage --path-coverage-graph-mode=abstracted \
 	modbat.examples.NioSocket1
 savemv modbat.examples.NioSocket1-root-StateGraph.dot nio-path-state-1000.dot
 checkfile nio-path-state-1000.dot
 savemv modbat.examples.NioSocket1-root-PointGraph.dot nio-path-point-1000.dot
 checkfile nio-path-point-1000.dot
 
+## heuristic search 10 test cases
+run 0 $APP -n=10 -s=1 --no-redirect-out --log-level=debug --dotify-path-coverage --search=heur \
+	modbat.examples.NioSocket1
+savemv modbat.examples.NioSocket1-root-StateGraph.dot nio-path-state-heur-10.dot
+checkfile nio-path-state-heur-10.dot
+savemv modbat.examples.NioSocket1-root-PointGraph.dot nio-path-point-heur-10.dot
+checkfile nio-path-point-heur-10.dot
+## heuristic search 20 test cases
+run 0 $APP -n=20 -s=1 --no-redirect-out --log-level=debug --dotify-path-coverage --search=heur \
```

Comment:
Thank you for adding so many tests.
Is there something that changes in particular in the output for n=20, 50, 150, 200?
If not, we can probably remove some of these tests. It is easier to maintain two or three variants than so many.